docs/resources/job.md

```diff
@@ -3,63 +3,45 @@
 page_title: "qovery_job Resource - terraform-provider-qovery"
 subcategory: ""
 description: |-
-  Provides a Qovery job resource. This can be used to create and manage Qovery job registry.
+  Provides a Qovery job resource. This can be used to create and manage Qovery job registry. Cron job example https://github.com/Qovery/terraform-examples/tree/main/examples/deploy-a-cron-job Lifecycle job example https://github.com/Qovery/terraform-examples/tree/main/examples/deploy-a-lifecycle-job
```

Comment:
we can move the examples in the example of usage section , like "For a complete example, have a look at this .."

Reply:
I don't find a way to do it... All the content of markdown files is generated by Terraform docs task.